package.json

```diff
@@ -9,6 +9,7 @@
     "lib"
   ],
   "scripts": {
+    "precommit": "npm run test",
```

Comment:
Also, it's best to run ava prepush too, Since if the tests fail the PR is not accepted anyway. You can use '--no-verify' to bypass it if Needed.